site/src/components/common/Button.tsx

```diff
@@ -0,0 +1,122 @@
+import { forwardRef, HTMLAttributes, ReactNode, RefObject } from "react";
+import styled, { css } from "styled-components";
+
+type ButtonVariant = "Contained" | "Outlined" | "Text";
```

Comment:
We always define values like this in `camelCase`, this should probably be the same here. 

```suggestion
type ButtonVariant = "contained" | "outlined" | "text";
```

Reply:
Since we use `camelCase` for everything else like this in the site, it should be the same here imo, even if it requires additional mapping. 

@johnnyomair FYI: This is one of the reasons I don't like enums in combination with our coding-guidelines, requiring enums to be `PascalCase` 😄